.github/workflows/python_package.yml

```diff
@@ -79,29 +80,57 @@ jobs:
         include:
           - os: macos-13
             task: regular
+            compiler: clang
+            python_version: '3.11'
+          - os: macos-13
```

Comment:
`macos-13` is going to close down soon (cf. https://github.blog/changelog/2025-09-19-github-actions-macos-13-runner-image-is-closing-down/). Can we simply use `macos-15` here (arm64 instead of x86-64). We can also do that in a follow-up PR though

Reply:
Thank you, I've been meaning to write up an issue about that. Let's do it in a follow-up PR.